packages/viewer/src/components/viewer-transform-widget/viewer-transform-widget.spec.tsx

```diff
@@ -251,6 +250,8 @@ describe('vertex-viewer-transform-widget', () => {
 
     window.dispatchEvent(new MouseEvent('pointermove'));
 
+    await new Promise((resolve) => setTimeout(resolve, 75));
```

Comment:
The jest functions around mocking timers were not working for these tests.

Reply:
Minor nit: We've actually got a [small utility](https://github.com/Vertexvis/vertex-web-sdk/blob/master/packages/utils/src/async.ts#L23) for this you could use 🙂